config/alphagov.yml

```diff
@@ -184,24 +184,27 @@ reliability-engineering:
   quotes:
     - "Someone needs to think of some new quotes!"
 
-taxonomy:
+taxonomy-and-naviation:
```

Comment:
@cbaines there's a typo here. Perhaps rename it to `taxonomy-and-nav` for consistency (might fix `seal me up`)